src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs

```diff
@@ -10,48 +10,97 @@
 
 namespace System
```

Comment:
Did you generate this via the GenAPI's target? Part of looks that way but other parts, like the using statements, don't. I would suggest generating by running the GenerateReferenceSource target https://github.com/dotnet/corefx/blob/master/dir.targets#L113.